pkgs/top-level/python-packages.nix

```diff
@@ -5295,6 +5295,25 @@ in {
     };
   };
 
+  isso = buildPythonPackage rec {
```

Comment:
This is an application and should therefore be called directly from `all-packages.nix`, and not via `python-packages.nix`.

Reply:
is it ok if I do it like this?
https://github.com/NixOS/nixpkgs/pull/13587/files#diff-036410e9211b4336186fc613f7200b12R9553

Reply:
Or should I move the whole buildPythonPackage in its own file in pkgs/applications (or pkgs/servers)?

Reply:
Yes, please create a `default.nix` in `pkgs/servers/isso/`, and remember to use `buildPythonApplication`.
However, `misaka` (latest version) needs to be put in `pkgs/development/python-modules` and called from `python-packages.nix`.
You can then use `overridePytonAttrs` in isso's `default.nix` to specify a custom misaka version.